plugins/source/aws/resources/services/fsx/data_repo_tasks.hcl

```diff
@@ -0,0 +1,48 @@
+//check-for-changes
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
+resource "aws" "fsx" "data_repo_tasks" {
+  path = "github.com/aws/aws-sdk-go-v2/service/fsx/types.DataRepositoryTask"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreAccessDeniedServiceDisabled"
```

Comment:
Could this maybe be `IgnoreCommonErrors`?

Reply:
Updated, thanks!